emmaa_service/api.py

```diff
@@ -1774,209 +2022,265 @@ def list_curations(stmt_hash, src_hash):
-        selected format.
-    """
-    model = request.json.get('model')
-    if not model:
-        abort(Response('Need to provide model', 404))
-    date_format = request.json.get('date_format', 'datetime')
-    test_corpus = request.json.get('test_corpus', _default_test(model))
-    date = get_latest_available_date(
-        model, test_corpus, date_format=date_format, bucket=EMMAA_BUCKET_NAME)
-    return jsonify({'date': date})
+        return {'statements': stmt_jsons, 'link': link}
 
 
-@app.route('/demos')
-@jwt_optional
-def get_demos_page():
-    user, roles = resolve_auth(dict(request.args))
-    user_email = user.email if user else ""
-    return render_template('demos_template.html', link_list=link_list,
-                           user_email=user_email)
-
-
-@app.route('/models', methods=['GET', 'POST'])
-def get_models():
-    """Get a list of all available models."""
-    model_meta_data = _get_model_meta_data()
-    models = [model for (model, config) in model_meta_data]
-    return {'models': models}
-
-
-@app.route('/model_info/<model>', methods=['GET', 'POST'])
-def get_model_info(model):
-    """Get metadata for model."""
-    config = get_model_config(model)
-    info = {'name': model,
-            'human_readable_name': config.get('human_readable_name'),
-            'description': config.get('description')}
-    if 'ndex' in config:
-        info['ndex'] = config['ndex'].get('network')
-    if 'twitter_link' in config:
-        info['twitter'] = config['twitter_link']
-    return info
-
-
-@app.route('/test_corpora/<model>', methods=['GET', 'POST'])
-def get_tests(model):
-    """Get a list of available test corpora for model."""
-    tests = _get_test_corpora(model)
-    return {'test_corpora': list(tests)}
-
-
-@app.route('/tests_info/<test_corpus>', methods=['GET', 'POST'])
-def get_tests_info(test_corpus):
-    """Get test corpus metadata."""
-    model_meta_data = _get_model_meta_data()
-    for (model, config) in model_meta_data:
+@latest_ns.param('model', 'Name of EMMAA model, e.g. aml, covid19, etc.')
+@latest_ns.expect(url_parser)
+@latest_ns.route('/statements_url/<model>')
+class LatestStatementsUrl(Resource):
+    @latest_ns.marshal_with(link_model)
+    def get(self, model):
+        """Return a link to model latest statements file on S3."""
+        dated = request.args.get('dated', type=inputs.boolean)
+        fkey = None
+        if dated:
+            prefix = f'assembled/{model}/statements_'
+            fkey = find_latest_s3_file(EMMAA_BUCKET_NAME, prefix, '.gz')
+        else:
+            if does_exist(EMMAA_BUCKET_NAME,
+                          f'assembled/{model}/latest_statements_{model}'):
+                fkey = f'assembled/{model}/latest_statements_{model}.json'
+        if fkey:
+            link = f'https://{EMMAA_BUCKET_NAME}.s3.amazonaws.com/{fkey}'
+        else:
+            link = None
+        return {'link': link}
+
+
+@latest_ns.expect(date_parser)
+@latest_ns.route('/stats_date')
```

Comment:
To make this more similar to the other `latest` endpoints, should we add the model argument as `/stats_data/<model>` here?